tests/test_adjoint_jacobian.py

```diff
@@ -199,6 +206,8 @@ def test_pauli_rotation_gradient(self, stateprep, G, theta, dev):
     def test_Rot_gradient(self, stateprep, theta, dev):
         """Tests that the device gradient of an arbitrary Euler-angle-parameterized gate is
         correct."""
+        if device_name == "lightning.gpu" and dev.R_DTYPE == np.float32:
+            pytest.skip("Skipped FP32 tests for expval in lightning.gpu")
```

Comment:
Why doesn't this work? We should be able to use FP32 with LightningGpu too, now? Same with all the skips below.

Reply:
Fixed!